cg/meta/workflow/fastq.py

```diff
@@ -147,22 +147,45 @@ def parse_header(line: str) -> dict:
 
         return fastq_meta
 
+    @staticmethod
+    def parse_nanopore_header(line: str) -> dict:
+        fastq_meta = {"lane": None, "flowcell": None}
+        header_metadata: list = line.split(" ")
+        lane = header_metadata[3].split("=")
+        fastq_meta["lane"] = lane[1]
+        flowcell = header_metadata[5].split("=")
+        fastq_meta["flowcell"] = flowcell[1]
+        return fastq_meta
+
     @staticmethod
     def parse_file_data(fastq_path: Path) -> dict:
```

Comment:
Is there a way to separate this more into the microsalt workflow? I think including nanopore directly into a function we use for every fastq file makes it a bit harder to understand for someone unfamiliar to it? Maybe the microsalt analysis_api can have its own `gather_file_metadata_for_sample` function with the nanopore exception?

Reply:
I realised that there is a mutanfastqhandler lower down in the same file. I think these changes should be placed there.

Reply:
I have moved some to the mutantfastqhandler, but code is very re-written now (had to to make it work), however not all code is under mutantfastqhandler (for cases where I wrote a new function not present before). What do you think?